### PR TITLE
Add state saving

### PR DESCRIPTION
**Pull request by AndrKriv**

### CODE REVIEW

File: app/src/main/java/com/innowisegroup/datetimepicker/DatePickerFragment.kt [hunk 1]
```diff
@@ -125,6 +131,13 @@ class DatePickerFragment : Fragment() {
             Bundle().apply { putString(UPDATE_DATE_TAB_TITLE_KEY, newValue.formatDate()) })
     }
 
+    override fun onSaveInstanceState(outState: Bundle) {
+        super.onSaveInstanceState(outState)
+        outState.apply {
+            putSerializable(SELECTED_DATE, localDate)
+        }
```
Comment (dmorskoi) [inline app/src/main/java/com/innowisegroup/datetimepicker/DatePickerFragment.kt]: Do you need "apply" scope function here?

File: app/src/main/java/com/innowisegroup/datetimepicker/TimePickerFragment.kt [hunk 2]
```diff
@@ -87,12 +92,20 @@ class TimePickerFragment : Fragment() {
             Bundle().apply { putString(UPDATE_TIME_TAB_TITLE_KEY, newValue.formatTime()) })
     }
 
+    override fun onSaveInstanceState(outState: Bundle) {
+        super.onSaveInstanceState(outState)
+        outState.apply {
```
Comment (dmorskoi) [inline app/src/main/java/com/innowisegroup/datetimepicker/TimePickerFragment.kt]: Do you need "apply" scope function here?

**Review by Bwonner — APPROVED** [review]

File: app/src/main/java/com/innowisegroup/datetimepicker/DatePickerFragment.kt [hunk 3]
```diff
@@ -125,6 +131,13 @@ class DatePickerFragment : Fragment() {
             Bundle().apply { putString(UPDATE_DATE_TAB_TITLE_KEY, newValue.formatDate()) })
```
Comment (dmorskoi) [inline app/src/main/java/com/innowisegroup/datetimepicker/DatePickerFragment.kt]: Do you need here "apply" function?

Also change in 123 line to - "day ?: return"

Comment (Bwonner) [reply app/src/main/java/com/innowisegroup/datetimepicker/DatePickerFragment.kt]: Why not?

**Review by dmorskoi — APPROVED** [review]

**Pull request merged**